Integrations/SecurityAdvisor/SecurityAdvisor.py

```diff
@@ -0,0 +1,120 @@
+import demistomock as demisto
+from CommonServerPython import *
+import requests
+import json
+import collections
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+BASE_URL = demisto.params().get('url', 'https://www.securityadvisor.io')
```

Comment:
please reformat the integration according to our new code conventions
https://github.com/demisto/content/blob/master/docs/code_conventions/README.MD

Reply:
This is done.